.github/workflows/deploy-integ.yml

```diff
@@ -40,6 +40,14 @@ jobs:
         run: ./scripts/install.sh
       - name: Build all packages
         run: ./scripts/build-all-packages.sh
+      - name: Run unit tests
```

Comment:
Find it odd that we are running unit tests here

Reply:
Created a new workflow